services/core-api/app/cli_commands/prepare_data.py

```diff
@@ -0,0 +1,285 @@
+import csv
+import io
+import logging
+from datetime import datetime
+from pathlib import Path
+
+from app.api.mines.mine.models.mine import Mine
+from app.api.mines.permits.permit.models.mine_permit_xref import MinePermitXref
+from app.api.mines.permits.permit.models.permit import Permit
+from app.api.mines.permits.permit_amendment.models.permit_amendment import (
+    PermitAmendment,
+)
+from app.api.mines.permits.permit_amendment.models.permit_amendment_document import (
+    PermitAmendmentDocument,
+)
+from app.api.services.document_manager_service import DocumentManagerService
+from app.extensions import db
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+class ImportMinesAndPermits:
+    # Utility script to import mines, permits, amendments, and associated documents from a CSV file.
+    # It will take the documents specified and upload them to S3 using the document manager.
+    # The purpose for this script is to test out permit condition search in a real-ish environment with actual data.
+    #
+    # WARNING: DO NOT RUN THIS IN PRODUCTION
```

Comment:
Juuust to be extra-safe, could we do a `if Config.ENVIRONMENT_NAME === 'production' : throw new Error(NO! Not allowed!)`